scripts/create_policy

```diff
@@ -242,19 +446,29 @@ def main():
             policy["ima"]["ignored_keyrings"] = ignored_keyrings
             policy["verification-keys"] = base_policy.get("verification-keys", "")
         except (PermissionError, FileNotFoundError) as ex:
-            print(f"An error occurred while loading the policy: {ex}")
+            print(f"An error occurred while loading the policy: {ex}", file=sys.stderr)
             ret = 1
         except json.decoder.JSONDecodeError as ex:
-            print(f"An error occurred while converting the policy to a JSON object: {ex}")
+            print(f"An error occurred while converting the policy to a JSON object: {ex}", file=sys.stderr)
             ret = 1
     if ret:
         sys.exit(ret)
 
-    # Add the digests map either from the allowlist, if given, or the IMA measurement list.
+    if (args.local_repo or args.remote_repo) and not HAS_RPM:
+        print('To analyze RPM repositories the "rpm" Python module is required', file=sys.stderr)
+        sys.exit(1)
+
+    # Add the digests map either from the allowlist, a repo, or the
+    # IMA measurement list.
     if args.allowlist:
-        policy["digests"], ret = process_flat_allowlist(args.allowlist, policy["digests"])
+        policy["digests"], ret = process_flat_allowlist(args.allowlist)
```

Comment:
This is not the same as before... If the tool read some sort of base policy then this here should merge new values into it - at least that's what was the original intention. Now it's a replacement of the contents of the base policy. Both operations are probably valid, so this is a bit  'tricky'  on how to give users this type of control over it without the tool becoming confusing to use.

Reply:
Reverting this then. Thanks!